### PR TITLE
fix(test): remove use-rowid key in test/*/conf/*.toml

### LINKED ISSUE #2700: integration_test fail for ticdc v5.2.0
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- 1. download ticdc master(v5.2.0)
- 2. run "make"
- 3. check and prepare the related dependent binary
- 4. run "make integration_test". 

2. What did you expect to see?
run integration successfully

3. What did you see instead?
integration test fail

```
[2021/09/01 11:30:32.939 +08:00] [ERROR] [main.go:38] ["parse cmd flags"] [error="unknown keys in config file /home/maxshuang/remote/ticdc/tests/autorandom/conf/diff_config.toml: [use-rowid]"] [errorVerbose="unknown keys in config file /home/maxshuang/remote/ticdc/tests/autorandom/conf/diff_config.toml: [use-rowid]
```

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        v5.2.0
        ```

    - TiCDC version (execute `cdc version`):

        ```
        v5.2.0
        ```

     - sync_diff_inspector version (execute `cdc version`):
     
        ```
        v5.2.0
        ```

### PR DESCRIPTION
**Pull request by maxshuang**
new version sync_diff_inspector v5.2.0 can not recognize the use-rowid key, and will cause integration fail. Since this key is not used in low version, just comment it

Closes #2700

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### Release note 

```release-note
None
```

### CODE REVIEW

File: tests/autorandom/conf/diff_config.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ log-level = "info"
 chunk-size = 10
 check-thread-count = 4
 sample-percent = 100
-use-rowid = false
+# use-rowid = false
```
Comment (amyangfei) [inline tests/autorandom/conf/diff_config.toml:7]: What about removing this line directly

**Review by amyangfei — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**